oresat_star_tracker/beast_test.py

```diff
@@ -0,0 +1,81 @@
+
+import unittest
+import sys
+
+# Needs to be fixed
+# sys.path.append('.')
+from os.path import abspath, dirname
+
+from .solver import Solver, SolverError
+from .beast import beast
+
+import datetime
+from  datetime import datetime
```

Comment:
Seperate import int 3 sections: internal python, external libs and, then internal project

Reply:
Also, `import x`  before `from x import  y`